vault/resource_mfa_okta_test.go

```diff
@@ -0,0 +1,50 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestMFAOktaBasic(t *testing.T) {
+	mfaOktaPath := acctest.RandomWithPrefix("mfa-okta")
+	resourceName := "vault_mfa_okta.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testutil.TestEntPreCheck(t) },
+		Providers: testProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testMFAOktaConfig(mfaOktaPath),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "name", mfaOktaPath),
+					resource.TestCheckResourceAttr(resourceName, "username_format", "user@example.com"),
+					resource.TestCheckResourceAttr(resourceName, "org_name", "hashicorp"),
+				),
+			},
+		},
+	})
+}
+
+func testMFAOktaConfig(path string) string {
+	userPassPath := acctest.RandomWithPrefix("userpass")
```

Comment:
nit: only referenced once, so maybe inline it in your call to `fmt.Sprintf()` below.

Reply:
Fixed in 5d506d7